packages/doltgres/content/concepts/sql/procedures.md

```diff
@@ -6,16 +6,16 @@ title: Stored Procedures
 
 ## What is a Stored Procedure?
 
-A stored procedure is SQL code that can be accessed using SQL `SELECT` syntax. Much like a function in
-other programming languages, you can pass values into a stored procedures. Results are returned as a
-table.
+A stored procedure is SQL code that can be accessed using SQL `CALL` syntax. Much like a function in
+other programming languages, you can pass values into a stored procedures. Stored procedures cannot
+return results, unlike functions.
 
 Database users create procedures. Procedures are schema and are stored along with other schema
 elements in the database.
 
 ## Doltgres support for Stored Procedures
 
 Doltgres comes with [many built-in stored procedures for version control
```

Comment:
I think this sentence needs to be removed and we need a function heading.

Reply:
For now I just added the function heading.